### PR TITLE
PM-12662 - Mass Update App Images (Part 1 of 2)

### PR DESCRIPTION
**Pull request by phil-livefront**
## 🎟️ Tracking

[PM-12662](https://bitwarden.atlassian.net/browse/PM-12662)

## 📔 Objective

- This update only includes the `Images` folder as part of the redesign work. The icons will be included in the next PR. I kept them separate to minimize the size of the changes.
- Confirmed with design that we are supposed to remove the expired link image.

## 📸 Screenshots

- Please see snapshots, there are quite a lot of changes. 

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[PM-12662]: https://bitwarden.atlassian.net/browse/PM-12662?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: BitwardenShared/UI/Platform/Settings/Settings/AccountSecurity/PendingRequests/__Snapshots__/PendingRequestsViewTests/test_snapshot_empty.1.png
Comment (matt-livefront) [inline BitwardenShared/UI/Platform/Settings/Settings/AccountSecurity/PendingRequests/__Snapshots__/PendingRequestsViewTests/test_snapshot_empty.1.png:1]: ❓ Can you check the size on this? It looks large compared to some of the other empty states.

File: BitwardenShared/UI/Auth/Login/TwoFactorAuth/__Snapshots__/TwoFactorAuthViewTests/test_snapshot_default_yubikey.1.png
Comment (matt-livefront) [inline BitwardenShared/UI/Auth/Login/TwoFactorAuth/__Snapshots__/TwoFactorAuthViewTests/test_snapshot_default_yubikey.1.png:1]: ❓ This also seems large compared to the designs.

File: BitwardenShared/UI/Platform/Settings/Settings/AccountSecurity/PendingRequests/__Snapshots__/PendingRequestsViewTests/test_snapshot_empty.1.png
Comment (phil-livefront) [reply BitwardenShared/UI/Platform/Settings/Settings/AccountSecurity/PendingRequests/__Snapshots__/PendingRequestsViewTests/test_snapshot_empty.1.png:1]: sure thing!  good catch...

File: BitwardenShared/UI/Auth/Login/TwoFactorAuth/__Snapshots__/TwoFactorAuthViewTests/test_snapshot_default_yubikey.1.png
Comment (phil-livefront) [reply BitwardenShared/UI/Auth/Login/TwoFactorAuth/__Snapshots__/TwoFactorAuthViewTests/test_snapshot_default_yubikey.1.png:1]: yeah, i double check this for sure.

**Review by matt-livefront — DISMISSED** [review]

**Review by matt-livefront — APPROVED** [review]

**Pull request merged**